src/__main__.py

```diff
@@ -1,4 +0,0 @@
-from main import main
```

Comment:
Should we delete the __main__.py file? I used it just to have one nice and small executable file instead of a hugh mungus script. Just for look. Thanks!

Reply:
Yeah I've never seen it done like this before. It's generally better to do it this way to develop a habit so when you look at other peoples' code you won't be surprised.

Reply:
Yeah, I thought about it too. Nice edit! Will have just one main.py file.
I am thinking about reallocating functions connected with the message reply yo message.py
Waiting to merge the pull request!